res/mock_evm/src/lib.rs

```diff
@@ -8,6 +8,7 @@ use near_sdk::{near_bindgen, PanicOnDefault};
 pub mod ft;
 mod metadata;
 mod out;
+mod selfcall;
```

Comment:
Just out of curiosity. What's the point of the such name?

Reply:
All the modules have names describing the types of methods they define. `ft` for FT calls, metadata for view methods. `self` is unfortunately not possible since it's a keyword and can't be used as an identifier. Open to better naming.

Reply:
Maybe `private`?

Reply:
As for me, it's a weird classification. What I suggest is all aurora-eve methods to put into the `lib.rs` and methods related to the `eth-connector` to move into new created `etc_connector.rs` module. `ft.rs` and `metadata.rs` just remove.

Reply:
I was basically following the existing convention but I'm happy with your proposed solution as well. I've moved over the self calls to lib.rs and in a separate PR we can move the rest of the stuff over.

I guess I don't know much yet but isn't the eth-connector separate from the engine? Should the two live in the same mock-evm crate?

Reply:
I think it should because after merging [Split NEP-141](https://github.com/aurora-is-near/aurora-engine/pull/607) we will communicate with `aurora-eth-connector` smart contract via cross-contract calls for a while. So, `aurora evm` will be like a proxy for `aurora-eth-connector`. It means `aurora-evm` will have eth-connector's methods. @mrLSD correct me if I am wrong.

Reply:
@mandreyel current mock is ok.
@aleksuss no it is not. Eth-connector method after some period of deprecation (like 3 months) will be removed from the Engine. But I'll do things related to the new eth-connector by myself. Don't worry.

Reply:
I think it can be renamed like "inner_call"